packages/application-components/src/components/portals-container/portals-container.tsx

```diff
@@ -162,41 +159,42 @@ const PortalsContainer = forwardRef<TLayoutRefs, TPortalsContainerProps>(
       }
     );
 
+    const isModalOpen = stackingLayers.length > 0;
+
     return (
       <>
         <Global
-          // Apply some global styles, based on the `.ReactModal__Body--open` class.
           styles={[
-            css`
-              .ReactModal__Body--open
+            isModalOpen &&
+              css`
```

Comment:
Previously, the `.ReactModal__Body--open` class was added to the root element. Now, global styles are keyed off the stacking layers count.